app/src/ui/branches/branches-container.tsx

```diff
@@ -65,17 +67,23 @@ export class BranchesContainer extends React.Component<
     }
   }
 
-  private onFilterKeyDown = (event: React.KeyboardEvent<HTMLInputElement>) => {
+  private shouldCloseFoldout = (when: () => boolean) => (
```

Comment:
Sorry, I actually meant the `when` parameter here, not the function name 🤦‍♂️ 

`onFilterKeyDown` here was fine.

Reply:
Where's the Homer Simpson Doh! when you need it?! 😆